.github/ISSUE_TEMPLATE/system-team-qg-checklist.md

```diff
@@ -22,159 +22,73 @@ QG5 Approval: yes/no <!-- Note: Please ask for the approval from Release Managem
 
 ### Check of Tractus-X Release Guidelines
 
-This QG x Check is depending on the mandatory information from our current [Release Guidelines](https://eclipse-tractusx.github.io/docs/release).
+- Currently implemented automatic checks can be found under your product on our [Release Guidelines Checks Board](https://eclipse-tractusx.github.io/sig-release/) 
+- This QG x Check is depending on the mandatory information from our current [Release Guidelines](https://eclipse-tractusx.github.io/docs/release)
```

Comment:
would remove the 'x' after QG

Reply:
[chore: fix typo + reduce x + removed PRERELEASE](https://github.com/eclipse-tractusx/.github/pull/15/commits/5577a78c452beadae1d5b7887dd6fc0931164617)